File: 프로그래머스/lv1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.java
import java.util.Arrays;

class Solution {
    public int[] solution(int[] arr, int divisor) {
        int[] answer = new int[arr.length];
        int index = 0;
        for(int i = 0; i < arr.length; i++) {
            if (arr[i] % divisor == 0){
                answer[index++] = arr[i];
            }
        }
        if (index == 0){
            return new int[]{-1};
        }
        answer = Arrays.copyOfRange(answer, 0, index);
        Arrays.sort(answer);

        return answer;
    }
}

File: JavaScript/프로그래머스/1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.js
function solution(arr, divisor) {
    let answer= [];
    arr.forEach(el => {
        if (el % divisor == 0) {
            answer.push(el);
        }
    });
    return (answer.length > 0) ? answer.sort((a, b) => a - b) : [-1];
}